resources/checks/checks.yml

```diff
@@ -472,6 +482,8 @@ post:
         to.setAddressState("CA");
         to.setAddressZip("94107");
 
+        UseType use_type = new UseType("operational");
```

Comment:
@BennyKitchell am I supposed to pass the use_type in?

Reply:
@rogue0137 I would remove this line, and just pass in the string into line 498. When you declare the object, it's properties are already declared internally.